READMEs/erc20-enterprise.md

```diff
@@ -68,6 +68,19 @@ erc20_enterprise_token = erc721_nft.create_datatoken(
 )
 print(f"ERC20 Enterprise address: {erc20_enterprise_token.address}")
 
+```
+Then, please refer to [publish-flow](publish-flow.md) to generate your metadata and encrypted files.
+Asset creation will be based on the deployment of ERC20 Enterprise token like this:
```

Comment:
@trentmc I'm tagging you here for approval. I advised Maria to use this structure to avoid repeating herself. The publishing part is described in the publish-flow readme, which we recently introduced. Since the only change here is in the asset creation itself (one instruction differs), while the metadata and encrypted files are identical to the publish flow, I advised against redoing the whole thing and using this language instead.

I think it's the right call, but an OK from you is probably warranted. Thanks!